scrapers/Carnalplus.yml

```diff
@@ -0,0 +1,39 @@
+name: CarnalPlus
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - carnalplus.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //html/head/title
+        postProcess:
+          - replace:
+              - regex: \| CarnalPlus.com
+                with:
+      Date:
+        selector: //div[@class='releasedate']
+        postProcess:
+          - replace:
+              - regex: \|[^|]*$
+                with:
+          - parseDate: January 02, 2006
+      Image:
+        selector: //script[contains(text(),"useimage")]
+        postProcess:
+          - replace:
+              - regex: '.*useimage\s=\s"([^"]+).+'
+                with: https://carnalplus.com$1
```

Comment:
This isn't working to get the image - I suggest using `selector: //video/@poster` without the `postProcess` to get the image. This works locally for me.